canonical/jaas-dashboard
# WD-2773 - Fix additional model access

## Done

- Fix additional models so they can be accessed.
- Get the correct user when connecting to additional models.

## QA

- For this you will need two local controllers (e.g. two multipass instances each running a controller).
- Configure your dashboard to use one of the controllers.
- Go to the controller list.
- Click the 'Register a controller button' and add the second controller.
- Go to the model list and make sure you can view the model details for models on both controllers.
- Go back to the model list and click the access button and make sure you can add/remove users for models on both controllers.

## Details

Fixes: #1066.
https://warthogs.atlassian.net/browse/WD-2773
